scripts/mender-layers.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
It's not clear to me from the commit text why any of the files in `scripts` should be removed.